src/Neo.VM/JumpTable/JumpTable.Splice.cs

```diff
@@ -95,13 +95,22 @@ public virtual void SubStr(ExecutionEngine engine, Instruction instruction)
         {
```

Comment:
@shargon why isn't there a vm limit in this?

Reply:
All the integers are limited, where is not?

Reply:
Because a substr can't be longer than the previous item.. theoretically